src/RA_Core.cpp

```diff
@@ -1238,10 +1239,18 @@ API void CCONV _RA_InvokeDialog(LPARAM nID)
             break;
 
         case IDM_RA_FILES_ACHIEVEMENTEDITOR:
-            if (g_AchievementEditorDialog.GetHWND() == nullptr)
+            if (::FindWindow(nullptr, _T("Achievement Editor")) == nullptr)
```

Comment:
why was this change made? does it need to be applied to line 1252 as well?

Reply:
Well I can change it back it you want it that way. I was trying to limit needing Globals and use an API call instead.

Reply:
FindWindow is an API call that has some amount of overhead for iterating through the windows on the system and doing strcmps on their titles. the previous implementation (albeit a global) was a fixed integer reference and required no overhead.

This code is triggered by selecting a menu option, so performance probably isn't key, but this change doesn't feel like an improvement and definitely isn't related to changing the RapidJSON version.

I'm fine with the pre-populating an achievement title, but you do _four_ ::FindWindow calls to get there.

Reply:
alright, I'm gonna close this and branch out.